app/components/events/view/export/download-zip.js

```diff
@@ -3,4 +3,9 @@ import Ember from 'ember';
 const { Component } = Ember;
 
 export default Component.extend({
+  actions: {
+    startGeneration() {
```

Comment:
No ... Remove this action here in the component and in your component hbs file, instead of `{{action 'startGeneration'}}` just do `{{action startGeneration}}`